### ISSUE TITLE
Poll for future of ngx-daterangepicker-material

### ISSUE THREAD

**Issue by fetrarij** [fetrarij/ngx-daterangepicker-material#308]
Hello all,

As you noticed, the version 10 of angular/material  has now a daterange picker with, but recently with version 9 we released the version 3 of ngx-daterangepicker-materia wich depends on angular/material so my question is now, with theses options bellow, what do you recommend and why ?

/poll "Keep current version (v3) which depends on angular/material" "revive v2 (make it work with angular 10) which is enterely independant and you can style customise your style" "other recommendation?"

**Comment by fetrarij** [comment]
@dmpasilva @praveenpuglia @MichielMag @angelo510 @steveacalabro @veerritu @Timebutt @mateusmcordeiro @styopdev @kirankumarsripati @duxor @butterknight @batgos @XHotSniperX @yousefkhan @sampath3797 @kodevtech @kgalanop @dwjohnston 
hello, could you participate? thanks in advance

**Comment by MichielMag** [comment]
Hey Fetraji,

The DateRange picker of angular/material actually differs from ngx-daterangepicker as it has no time selection.
My choice would actually be number 2, to revive v2. We don't use any material styling and I've customised the style completely to be integrated with our application.

That being said, v3 is probably still a nice alternative for people who do wish for material styling as it has a timepicker which angular/material doesn't.

**Comment by kirankumarsripati** [comment]
Hi @fetrarij ,

We previously used ng2-daterangepicker which uses jquery & https://www.daterangepicker.com/ library. Also, it doesn't work well with forms. After we started using Angular Material in our application and to fix issues with forms, this library helped us to replace the previous one. Our application has grown into big and we stuck at version 6 as of now and using 2.1.9. As said by @MichielMag, we also customized the Calendar CSS to match our styling. It's ok for us to not to have material dropdown everywhere. So, my choice would actually be number 2, to revive v2.

**Comment by RtzS** [comment]
Hi @fetrarij ,

I am using V2.2.10 ngx-daterangepicker-material as my project is based on AG6. And this calendar is really helped me to  fulfill my purpose to add range picker in my project. As others mentioned I also worked on CSS customization but it is fine for me. So i would like to go with second option that please upgrade V2 version as in this version some latest features are not present. 

Thankyou :)

**Comment by butterknight** [comment]
Hi @fetrarij,

I would agree with the others. With the official Material component I think it makes more sense to go with v2 revival. Official component is missing some of the features from this library so I think it would still have a lot of value.

**Comment by sampath3797** [comment]
yes,  upgrade and support to ngx daterangepicker material is really
required.  I am using this in my project and it helped a lot for me.
it's really amazing datepicker.

**Comment by kgalanop** [comment]
Hello @fetrarij 

I would also go with option 2. 
Being independent from material would also provide the ability for more users to use this library without necessarily being forced to use material, let alone that currently it has more features than the corresponding material implementation and better UI and layout in my opinion.

**Comment by veerritu** [comment]
@fetrarij  Option 2 anytime as its good to keep it customisable.
It's easy to implement and use.

**Comment by kavvya97** [comment]
i would really like to continue using ngx-daterangepicker and i would like it more if it is customisable. Hence i am going with option 2.

**Comment by alexmacavei** [comment]
Please don't kill it, we just adopted it in a production app :) I really needed those weeknumbers and for some reason the Angular Material guys don't want to include it as it's not part of the material spec.

**Comment by revatim** [comment]
I would like to go with option 2 
It has better implementation, functionality and can use it in projects without being forced to use angular/material. Being able to customize it also a plus.

**Comment by sac-cas** [comment]
Option 2 would be my choice. Maybe replacing momentjs with luxon, dayjs or date-fns :)

**Comment by fetrarij** [comment]
Hi, just to let you all know that v5 is out and momentjs is replaced with dayjs

**Comment by veerritu** [comment]
That's great 😀.

I will update soon.

Regards,
Rituja
